_posts/2021-12-10-start-a-workspace-by-entering-a-url.adoc

```diff
@@ -0,0 +1,98 @@
+
+As a user, you can create a workspace containing a clone of a Git repository by entering a custom URL in the address bar of your browser.
+
+As a user, you can also create a workspace containing a clone of a Git repository by entering the URL of the Git repository on the {prod} dashboard page.
+
+=== Short URL syntax for cloning hosted Git repositories
+
+In the earlier versions of {prod}, the {prod-short} URL syntax began with `/f/?url=`, for example `{prod-url}/f/?url=https://github.com/my/repository`, but it was complex and error-prone. While that syntax is still supported, short syntax should be used as follows:
+
+To start a workspace with a clone of a GitHub, GitLab, or Bitbucket repository, go to your browser's address bar and enter the {prod-short} URL, prepended with a `#` character and appended with the repository link. Using the example of Eclipse Che hosted by Red Hat: link:https://workspaces.openshift.com#https://github.com/eclipse-che/che-server[]. Then {prod} starts a workspace and clones the repository, `\https://github.com/eclipse-che/che-server` in this example.
+
+
+{prod} also searches for a `.devfile.yaml` or `devfile.yaml` file in the given repository. The devfile describes the containers and commands to use for that project. Omitting this file will result in a default container, which provides different languages (including Java, Node.js, Python, PHP, Golang, etc) and tools (including `curl`, `jq`, `yq`, etc).
+
+For more information about devfiles, see the link:https://devfile.io/[Devfile Documentation].
+
+=== URL syntax by Git provider
+
+URL syntax for cloning GitHub-hosted repositories to the starting workspace:
+
+* To clone the default branch on GitHub to the starting workspace:
++
+`#https://github.com/_<user_or_org>_/_<repository>_`
++
+Example: `#https://github.com/eclipse-che/che-plugin-registry`
+
+* To clone a specified branch on GitHub to the starting workspace:
++
+`#https://github.com/_<user_or_org>_/_<repository>_/tree/_<branch-name>_`
++
+Example: `#https://github.com/eclipse-che/che-plugin-registry/tree/7.39.x`
+
+* To clone the branch of a pull request on GitHub to the starting workspace:
++
+`#https://github.com/_<user_or_org>_/_<repository>_/pull/_<pull-request-id>_`
++
+Example: `#https://github.com/eclipse-che/che-plugin-registry/pull/1097`
+
+URL syntax for cloning GitLab-hosted repositories:
+
+* To clone the default branch on GitLab to the starting workspace:
++
+`#https://gitlab.com/_<user_or_org>_/_<repository>_`
+
+* To clone a specified branch on GitLab to the starting workspace:
++
+`#https://gitlab.com/_<user_or_org>_/_<repository>_/-/tree/_<branch>_`
+
+
+=== Using a different editor or `IDE`
+
+The default IDE is a property of {prod}, and the default value is `eclipse/che-theia/latest`. See the docs for the link:https://www.eclipse.org/che/docs/che-7/installation-guide/advanced-configuration-options-for-the-che-server-component/#devfile-defaults[Devfile defaults].
```

Comment:
This sounds unnecessarily complicated:

> The default IDE is a property of {prod}, and the default value is `eclipse/che-theia/latest`. See the docs for the link:https://www.eclipse.org/che/docs/che-7/installation-guide/advanced-configuration-options-for-the-che-server-component/#devfile-defaults[Devfile defaults].

What about:

{prod} starts workspaces with a default IDE: {prod-short} Theia.

Reply:
the truth is that while default editor is theia, it can be using the next channel or stable channel depending on the installation

Reply:
I assume the less precise description is sufficient for all users who don't need a specific version. Am I correct?

Reply:
I'm fine